tests/requirements-py3.txt

```diff
@@ -0,0 +1,4 @@
+pytest>=2.6.0
+pytest-twisted
+testfixtures
+jmespath
```

Comment:
I wonder why is jmespath only a testng requirement - it is required for some of Scrapy parts to work